src/components/OSCALProperties.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { styled } from "@mui/material/styles";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableRow from "@mui/material/TableRow";
+
+const PropertiesTable = styled(TableContainer)(
+  ({ theme }) => `
+  margin-top: ${theme.spacing(2)};
+  display: flex;
+  flex-direction: column;
+  text-transform: capitalize;
+`
+);
+
+export default function OSCALProperties(props) {
```

Comment:
Is this really `OSCALProperties`? Like does this encapsulate what it means everywhere we have `Properties`? I do not think that every component needs its own file. This seems very specific to `SystemImplementation`. I'd argue the same for `InventoryItems`. 

This also has a lot of overlap with #590. We should choose one way to do it and then just fix it in the other pr.

Reply:
`props` is defined exactly the same throughout OSCAL. Some examples [profile/metadata/revisions/props](https://pages.nist.gov/OSCAL/reference/latest/profile/json-reference/#/profile/metadata/revisions/props), [plan-of-action-and-milestones/metadata/props](https://pages.nist.gov/OSCAL/reference/latest/plan-of-action-and-milestones/json-reference/#/plan-of-action-and-milestones/metadata/props), [system-security-plan/system-implementation/users/props](https://pages.nist.gov/OSCAL/reference/latest/system-security-plan/json-reference/#/system-security-plan/system-implementation/users/props). I think it would make sense to create a reusable component.

Reply:
In many of these contexts we will. Currently, we create tables for `system-characteristic/props` and `component/props` in our application, where this can be reused. Maybe not necessarily everywhere. Perhaps this be made into `OSCALPropertiesTable`.